contrib/packaging/bootupctl-update.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Update Bootloader on boot
+Documentation=https://github.com/coreos/bootupd
+ConditionPathExists=/dev/disk/by-label/EFI-SYSTEM
```

Comment:
We should add as well:
```
/dev/disk/by-partlabel/EFI\\x20System\\x20Partition
```

Reply:
There's already also `ConditionFirmware=uefi`

Reply:
That said we need to support BIOS too...so probably the best would be e.g.

```
RequiresMountsFor=/boot
ConditionPathExists=/boot/bootupd-state.json
```

Reply:
`ConditionPathExists=/boot/bootupd-state.json` 
wouldn't that prevent it to run on atomic desktops as bootupd was not enabled until now ?

Reply:
I am OK with `ConditionFirmware=uefi`, can remove it until we add `adopt-and-update` for BIOS later.
One minor suggestion, should we add `ConditionKernelCommandLine=ostree` to skip this for live boots like https://github.com/coreos/rpm-ostree/pull/4944 ?

Reply:
i think the overall agreement was to close this PR and carry this in the distribution